src/auth/flows.rs

```diff
@@ -1930,6 +2007,34 @@ pub async fn verify_email(
     }
 }
 
+#[post("email/subscribe")]
```

Comment:
we should have an email/unsubscribe as well (or a delete("email/subscribe"), or some way to unsubscribe)

Reply:
done through beehiiv